### ISSUE TITLE
swirlExpectation(condition(object)) error in lesson-4 (Vectors) in R Programming course

### ISSUE THREAD

**Issue by im2fast4u** [swirldev/swirl#461]
In R-Studio, at 16% completion mark in the lesson, I get this error. I tried uninstalling and reinstalling swirl package but it does not help:

| What do you think tf will look like?

1: a vector of 4 logical values
2: a single logical value

Selection: 1
Error in swirlExpectation(condition(object)) : 
  could not find function "is_equivalent_to"

| Leaving swirl now. Type swirl() to resume.

>

**Comment by seankross** [comment]
What do you get as the result of `packageVersion("testthat")`?

**Comment by jre7** [comment]
My students are having this problem as well but only those with Macs.

**Comment by seankross** [comment]
@jre7 make sure your students have swirl 2.4.1 and testthat 1.0.2 installed.

**Comment by im2fast4u** [comment]
I am getting 1.0.0 for testthat
Thank you Sean

**Comment by im2fast4u** [comment]
I reinstalled swirl one more time and let Swirl download R Programming course over the internet. After that I tried lesson 4 and did NOT face the above error. But still my version says 1.0.0 when I ran testthat. I have swirl 2.4.1

Earlier I was installing R Programming course from the swirl_courses-master.zip folder which was a few months old. I also re-downloaded that folder and did the above drill just to check, and it worked as well. 

Thank you
